tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -348,14 +394,20 @@ Replace data files for amendment
     user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}dates.csv
     user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}dates.meta.csv
     user clicks button  Upload data files
-    user checks headed table body row cell contains  Status          2  Complete   wait=180
+    user opens details dropdown  Footnote 2 seven filters
+    user clicks button  Delete footnote
+    user clicks button  Confirm
+    user reloads page
+    user scrolls down  2000
```

Comment:
I don't think this is necessary? If you need to scroll down to the button, I'd use `user scrolls to element` instead. But I think you should be able to just remove this.